sycl/source/detail/scheduler/commands.cpp

```diff
@@ -2010,7 +2010,6 @@ cl_int ExecCGCommand::enqueueImp() {
     ExecInterop->MInteropTask->call(InteropHandler);
     Plugin.call<PiApiKind::piEnqueueEventsWait>(MQueue->getHandleRef(), 0,
                                                 nullptr, &Event);
-    Plugin.call<PiApiKind::piQueueRelease>(MQueue->getHandleRef());
```

Comment:
Where is the correct instance of the call to "piQueueRelease"? Can you cover this by a test?

Reply:
Actually, getHandleRef() do only 1 thing -  returns native object, so
`Plugin.call<PiApiKind::piQueueRelease>(MQueue->getHandleRef());`  - is not correct, because it does **an extra piQueueRelease call** and that leads to a crash in the test [interop_task.cpp](https://github.com/intel/llvm-test-suite/blob/intel/SYCL/Basic/handler/interop_task.cpp) 

Before my patch, the line looked like this:
`Plugin.call<PiApiKind::piQueueRelease>(
        reinterpret_cast<pi_queue>(MQueue->get()));`
and, the fact is that get() method does 2 things: retains and returns the native object. Let's pay attention that we called release immediately for the returned object in the same line.

As a result, now, and before my patch, we have the correct reference count for the queue object, But before we did extra instructions in this line. I see no reason to add a new test.

Reply:
Makes sense (I didn't notice before that the `.get()` was changed to `.getHandleRef()`.